test/quantization/bc/test_backward_compatibility.py

```diff
@@ -216,6 +216,7 @@ def test_conv3d_relu(self):
             # TODO: graph mode quantized conv3d module
 
     @override_qengines
+    @unittest.skipIf(IS_AVX512_VNNI_SUPPORTED, "This test fails on Intel Cascade Lake CI machines. Ref: GH Issue 59098")
```

Comment:
Instead of skipping, we should just fix this test so that it works on vnni machines. cc @vkuzo . This seems related to reduce range.

Reply:
@dskhudia, I can try fixing it, but I don't have the background knowledge, so I may not succeed. Nevertheless, I'll look at the source-code & will try. From #46749, it seems @vkuzo has a plan for fixing it. Thanks!